gettingstarted/index.md

```diff
@@ -5,6 +5,67 @@ layout: page
 Getting Started
 ---------------
 
+### Command line generator
+
+The easiest way to begin using courier is via the standalone command line tool. The standalone tool
+generates type safe bindings for any courier-supported language. Assuming Java (JRE) is already
+installed on your system, you can download the binary from
+[Bintray](https://dl.bintray.com/coursera/generic/courier) and install it by doing the following:
+
+~~~sh
+cd /tmp && \
+wget https://dl.bintray.com/coursera/generic/courier && \
```

Comment:
Oops my machine (mac, el cap) didn't come with wget...I had to use `curl -L https://dl.bintray.com/coursera/generic/courier >> courier && \`